File: app/src/main/java/com/example/shopapp/HiApplication.java
package com.example.shopapp;

import com.alibaba.android.arouter.launcher.ARouter;
import com.example.common.ui.component.HiBaseApplication;
import com.example.library.log.HiLogConfig;
import com.example.library.log.HiLogManager;
import com.example.library.log.printer.HiConsolePrinter;

public class HiApplication extends HiBaseApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        if (BuildConfig.DEBUG) {
            ARouter.openLog();
            ARouter.openDebug();
        }

        ARouter.init(this);
    }
}

File: app/src/main/java/com/example/shopapp/HiApplication.kt
package com.example.shopapp

import com.alibaba.android.arouter.launcher.ARouter
import com.example.common.ui.component.HiBaseApplication
class HiApplication : HiBaseApplication() {
    override fun onCreate() {
        super.onCreate()
        if (BuildConfig.DEBUG) {
            ARouter.openLog()
            ARouter.openDebug()
        }
        //初始化ARouter路由组件
        ARouter.init(this)
    }
}